### ISSUE TITLE
benchmark-compare=NUM option without NUM set, runs as if no benchmark-compare was set

### ISSUE THREAD

**Issue by hcuche** [ionelmc/pytest-benchmark#168]
When I try to compare the last saved reference (using `--benchmark-autosave` or `--benchmark-save=some-name` options), with the current run (using `--benchmark-compare=` as explain in the [documentation](https://pytest-benchmark.readthedocs.io/en/latest/usage.html#commandline-options))

```
--benchmark-compare=NUM
 	Compare the current run against run NUM (or prefix of _id in elasticsearch) or the latest saved run if unspecified.
```
There is no comparison made, only a standard benchmark run.

I have the following tests/test_perf.py file
```python
def fake_print():
    print("Fake")

def test_perf_cortex(benchmark):
    benchmark(fake_print)
```

When I run `--benchmark-autosave` the file is well generated in .benchmarks/ :
```
$ python3 -m pytest --benchmark-autosave tests      
(...)                                                                                                                                 
$ ls -R .benchmarks                                                                                                                                                                              .benchmarks:
Linux-CPython-3.6-64bit/

.benchmarks/Linux-CPython-3.6-64bit:
0001_unversioned_20200417_145821.json
```

Then running the compare option using the saved **NUM** works fine:
```
$ python3 -m pytest --benchmark-compare=0001 tests/
```
but running without **NUM**, runs pytest as if no `--benchmark-compare` was set
```
$ python3 -m pytest --benchmark-compare= tests/
```

==================
Ubuntu 18.04
```
$ uname -a
Linux xxxxxx 4.15.0-96-generic #97-Ubuntu SMP Wed Apr 1 03:25:46 UTC 2020 x86_64 x86_64 x86_64 GNU/Linux
```

Python version and dependencies
```
python=3.6.9
pytest=5.4.1
pytest-benchmark=3.2.3
```

**Comment by hcuche** [comment]
The bug is not actually on NUM not set but if there is parameters after the option. I'll do another issues

**Comment by ionelmc** [comment]
Yes, this is a argparse quirk - not much we can do about it.

Btw, another way (and I think this is the practice you should adopt for consistent CLI experience): `python3 -m pytest --benchmark-compare= tests` (note the space)

**Comment by hcuche** [comment]
Hi, I just tried the command line you gave me, with the space between "--benchmark-compare=" and "tests" but this is not working either. 
I also tested ```python3 -m pytest tests --benchmark-compare=``` (note the equal at the end), it also do not work where the same command works without the equal.
It seems that if you put the equal you must add an ID to make it work.

I guessed it was a argparse quirk but wanted to be sure. It might be good to documents this in a trouble shouting or somewhere.
